src/raster/r.area.createweight/r_area_createweight.html

```diff
@@ -0,0 +1,468 @@
+with the parameter <b>basemap_b</b>, and/or a raster with continuous
+values (e.g. distance to the nearest road, school, hospital,
+university...) with the parameter <b>distance_to</b>. From these raster
+maps, the Random Forest's explanatory variables (i.e. covariates,
+predictors) are calculated.  For each of the categorical rasters
+(basemaps), the proportion of each of the classes is calculated. For the
+continuous raster map, the average values are calculated.
+
+<p>Once trained, the Random Forest model predicts weights in a raster
+grid. The spatial resolution of this output weighted grid should be
+specified via the <b>tile_size</b> parameter, and its name via the
+<b>output</b> parameter. The <b>tile_size</b> must be greater than
+the spatial resolution of <b>basemap_a</b>. It it considered good
+practice that the <b>tile_size</b> also be greater than the spatial
+resolution of the other basemaps and distance map. The extent of the
+output weighted grid is created using the extent of the spatial units,
+i.e. the <b>vector</b> parameter.
+
+<p>By default, all classes from the raster basemap(s) are taken into
+account. If the user want only specific classes to be taken into account
+within the Random Forest model, it is possible to provide an optional
+list of these classes with the <b>basemap_a_list</b> parameter (for
+basemap_a) or the <b>basemap_b_list</b> parameter (for basemap_b, if
+used).
+
+<p>The out-of-bag error (OOB) of the model is printed in the console
+and gives an indication about the internal accuracy of the model
+(cross-validation on training set at the spatial units level). The log
+of the Random Forest run (including the OOB error) is saved in a file
+whose path and name has to be specified via the <b>log_file</b>
+parameter. Using the <b>-f</b> flag, the log file will include extended
+information about the cross-validation for each set of parameters
+tested in the grid search.
+
+<p>Feature importances in the model are plotted in a graph, whose path
+and name has to be specified via the <b>plot</b> parameter. By default,
+class values are used. Optionally, class labels can be used as plot
+labels, using the flags <b>-a</b> and <b>-b</b>, for <b>basemap_a</b>
+and <b>basemap_b</b> respectively. If these flags are used, the raster
+should already contain the associated category labels. The module <a
+href="https://grass.osgeo.org/grass-stable/manuals/r.category.html">r.
+category</a> can be used to set category labels for a raster map. If the
+flag is selected, but the class label does not exist, the class value
+will be kept.
+
+<p>Parallel processing is supported. The number of cores to be used
+should be specified via the <b>n_jobs</b> parameter.
+
+<h2>NOTES</h2>
+
+<p>The module <a
+href="https://grass.osgeo.org/grass-stable/manuals/r.mask.html">
+r.mask</a> is used within the module. The user should first remove any
+masks. If the user wish to run the add-on on a subset of the data, the

```

Comment:
```suggestion
masks. If the user wishes to run the add-on on a subset of the data, the

```

Reply:
subset of the data means what here? a subset of the input vector with spatial units? a subset of input explanatory variables?

Reply:
This sentence is confusing and not needed since the next paragraph is clear enough I guess.